images/hub/cull_idle_servers.py

```diff
@@ -83,12 +89,17 @@ def cull_one(user, last_activity):
         if not user['server'] and not cull_users:
             # server not running and not culling users, nothing to do
             continue
+        if not user['last_activity']:
+            continue
         last_activity = parse_date(user['last_activity'])
         if last_activity < cull_limit:
+            if user['pending']:
```

Comment:
`pending` means that the state of the spawner is changing (from stopped to starting or the other way around)?